packages/ui/src/filter/filter-list.tsx

```diff
@@ -33,14 +33,27 @@ export function FilterList({
         <div className="flex grow flex-wrap gap-x-4 gap-y-2">
           <AnimatePresence>
             {activeFilters?.map(({ key, value }) => {
+              if (key === "loader") {
+                return (
+                  <motion.div
+                    key={`loader-${value}`}
+                    initial={{ opacity: 0, y: 4 }}
+                    animate={{ opacity: 1, y: 0 }}
+                    className="h-9 w-48 animate-pulse rounded-md border border-gray-200 bg-white"
+                  />
+                );
+              }
+
               const filter = filters.find((f) => f.key === key);
               if (!filter) {
                 throw new Error(
                   "Filter.List received an activeFilter without a corresponding filter",
                 );
               }
 
-              const option = filter.options?.find((o) => o.value === value);
+              const option = filter.options?.find(
+                (o) => o.value.toLowerCase() === value.toLowerCase(),
+              );
```

Comment:
@steven-tey what did you run into that required this change? I think it's breaking things for non-string values

Reply:
basically the LLM will sometimes generate stuff in lowercase, which is why we need to make the comparison in lowercase to account for that – perhaps there's a better alternative here?

Reply:
Restored this, just with extra `typeof` checks 👍